src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/RolesController.kt

```diff
@@ -356,16 +357,16 @@ data class Role(
   @Schema(required = true, description = "Administration Type")
   val adminType: List<AdminTypeReturn>,
 ) {
-  constructor(r: Role) : this(
-    r.roleCode,
-    r.roleName,
-    r.roleDescription,
-    r.adminType
-  )
+  companion object {
+    fun fromDomain(role: Role): RoleDto =
+      with(role) {
+        return RoleDto(roleCode, roleName, roleDescription, adminType)
+      }
+  }
```

Comment:
I think this should stay as a constructor, passing in the Role to make the RoleDto